src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java

```diff
@@ -50,159 +50,159 @@ public class NotificationHandlerConfiguration {
 
```

Comment:
There is a huge diff here. can you undo the formatting change?

Reply:
Resolved it